misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/entity/CoverageReport.java

```diff
@@ -81,26 +82,20 @@ public CoverageReport(Module module) throws IOException {
      *
      * @throws IOException when file operations are failed
      */
-    public ModuleCoverage generateReport() throws IOException {
+    public ModuleCoverage generateReport(JarResolver jarResolver) throws IOException {
         String orgName = this.module.packageInstance().packageOrg().toString();
         String packageName = this.module.packageInstance().packageName().toString();
         String version = this.module.packageInstance().packageVersion().toString();
 
         Path moduleJarPath = target.cachesPath().resolve(orgName)
```

Comment:
Can't we get rid of the Target to resolve jar paths since we are using the jar resolver?

Reply:
Removed the use of target in this case